pkg/security/security_profile/dump/manager_test.go

```diff
@@ -0,0 +1,366 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build linux
+// +build linux
+
+package dump
+
+import (
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/DataDog/datadog-go/v5/statsd"
+
+	"github.com/DataDog/datadog-agent/pkg/security/config"
+)
+
+func compareListOfDumps(t *testing.T, out, expectedOut []*ActivityDump) {
```

Comment:
I think `t.Helper()` would help pinpoint the actual error position

Reply:
I actually just moved this file but somehow Git didn't pick up on the move for this one. Let keep the changes to a minimum ? The PR is already quite big :( But happy to make another PR with your fix if you want me to !